accounts/abi/bind/util_test.go

```diff
@@ -125,10 +125,7 @@ func TestWaitDeployedCornerCases(t *testing.T) {
 	// Create a transaction to an account.
 	code := "6060604052600a8060106000396000f360606040526008565b00"
 	tx := types.NewTransaction(0, common.HexToAddress("0x01"), big.NewInt(0), 3000000, gasPrice, common.FromHex(code))
-	tx, err := types.SignTx(tx, types.LatestSignerForChainID(big.NewInt(1337)), testKey)
-	if err != nil {
-		t.Fatalf("Failed to sign transaction: %s", err)
-	}
+	tx, _ = types.SignTx(tx, types.LatestSignerForChainID(big.NewInt(1337)), testKey)
```

Comment:
Why remove an error check in a test?

Reply:
Because it's not checked in the master branch 🤷  It feels like it's out of scope of libevm

Reply:
Why does it say

> Error Handling Improvements (already added in master)

though